test/e2e/app-dir/metadata-navigation/metadata-navigation.test.ts

```diff
@@ -33,14 +33,19 @@ describe('app dir - metadata navigation', () => {
 
       // TODO-APP: support render custom not-found in SSR for generateMetadata.
       // Check contains root not-found payload in flight response for now.
-      let hasRootNotFoundFlight = false
+      const flightDataPrefix = 'self.__next_f.push([1,"'
+      const flightDataSuffix = '"])'
+      let flightText = ''
       for (const el of $('script').toArray()) {
         const text = $(el).text()
-        if (text.includes('Local found boundary')) {
```

Comment:
This test assumed the entire text would be included in any given script chunk which is not the case. I've updated it to extract all the inliend flight data and then look for the result in the concatenated result